applications/plugins/VolumetricRendering/CMakeLists.txt

```diff
@@ -1,44 +1,35 @@
 cmake_minimum_required(VERSION 3.12)
-project(VolumetricRendering)
-
-set(VOLUMETRICRENDERING_MAJOR_VERSION 0)
-set(VOLUMETRICRENDERING_MINOR_VERSION 1)
-set(VOLUMETRICRENDERING_VERSION ${VOLUMETRICRENDERING_MAJOR_VERSION}.${VOLUMETRICRENDERING_MINOR_VERSION})
+project(VolumetricRendering VERSION 0.1)
 
 find_package(SofaFramework REQUIRED)
 find_package(SofaBase REQUIRED)
 find_package(SofaGeneral REQUIRED)
 find_package(SofaOpenglVisual REQUIRED)
 
 set(HEADER_FILES
-    initVolumetricRendering.h
-    OglTetrahedralModel.h
-    OglTetrahedralModel.inl
-    OglVolumetricModel.h
+    src/VolumetricRendering/config.h.in
+    src/VolumetricRendering/initVolumetricRendering.h
+    src/VolumetricRendering/OglTetrahedralModel.h
+    src/VolumetricRendering/OglTetrahedralModel.inl
+    src/VolumetricRendering/OglVolumetricModel.h
 )
 
 set(SOURCE_FILES
-    initVolumetricRendering.cpp
-    OglTetrahedralModel.cpp
-    OglVolumetricModel.cpp
+    src/VolumetricRendering/initVolumetricRendering.cpp
+    src/VolumetricRendering/OglTetrahedralModel.cpp
+    src/VolumetricRendering/OglVolumetricModel.cpp
 )
 
 add_library(${PROJECT_NAME} SHARED  ${HEADER_FILES} ${SOURCE_FILES} ${README_FILES})
 
-#Force includes to be like <VolumetricRendering/xxxxxx.h>
-target_include_directories(${PROJECT_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/..>")
-target_include_directories(${PROJECT_NAME} PUBLIC "$<INSTALL_INTERFACE:include>")
-
-set_target_properties(${PROJECT_NAME} PROPERTIES COMPILE_FLAGS "-DSOFA_BUILD_VOLUMETRICRENDERING")
```

Comment:
Why do you remove the preprocessor definition here ?

Reply:
Yes it can be removed, it is done through sofa_generate_package